mailbox/elasticsearch/src/main/java/org/apache/james/mailbox/elasticsearch/json/IndexableMessage.java

```diff
@@ -61,7 +61,6 @@ private static ZonedDateTime getSanitizedInternalDate(MailboxMessage message, Zo
         private IndexAttachments indexAttachments;
         private MailboxMessage message;
         private TextExtractor textExtractor;
-        private List<User> users;
```

Comment:
this was not needed for some permission checking?

Reply:
Not anymore: In multi-mailbox queries, James resolves the mailbox list the query should be run against Cassandra first, then blindly search these mailboxes in ES

BTW this 'users' projection is not maintained upon update and thus is dangerous.